### PR TITLE
uhd_fft: Added lo_offset correction to set_bandwidth command so we do…

### PR DESCRIPTION
**Pull request by ejk43**
… not filter out signal of interest

### CODE REVIEW

**Comment by ejk43** [comment]
Quick note, I dont currently have a usrp available to test, so this probably needs a check with hardware. I thought it could be a useful start anyway.

File: gr-uhd/apps/uhd_fft [hunk 1]
```diff
@@ -128,7 +129,7 @@ class uhd_fft(gr.top_block, Qt.QWidget, UHDApp):
         self.setup_usrp(uhd.usrp_source, args)
         self._ant_options = self.usrp.get_antennas(0)
         for c in range(len(self.channels)):
-            self.usrp.set_bandwidth(self.samp_rate, c)
+            self.usrp.set_bandwidth(self.samp_rate + abs(self.lo_offset), c)
```
Comment (mbr0wn) [inline gr-uhd/apps/uhd_fft]: Shouldn't this be `abs(self.lo_offset)`?

File: gr-uhd/apps/uhd_fft [hunk 2]
```diff
@@ -431,7 +432,7 @@ class uhd_fft(gr.top_block, Qt.QWidget, UHDApp):
         self.qtgui_waterfall_sink_x_0.set_frequency_range(self.freq, self.samp_rate)
         self.usrp.set_samp_rate(self.samp_rate)
         for c in range(len(self.channels)):
-            self.usrp.set_bandwidth(self.samp_rate, c)
+            self.usrp.set_bandwidth(self.samp_rate + abs(self.lo_offset), c)
```
Comment (mbr0wn) [inline gr-uhd/apps/uhd_fft]: Ditto

Comment (ejk43) [inline gr-uhd/apps/uhd_fft]: I realized we need abs() after making the original PR, so I added a second commit with abs(self.lo_offset). Do you see this line on your review? It shows up for me like this: 

`self.usrp.set_bandwidth(self.samp_rate + abs(self.lo_offset), c)`

If you dont see that, I'll try to find why the second commit doesnt show up in the PR. If you prefer, I can rebase and redo the PR... up to you.

**Comment by mbr0wn** [comment]
It now shows up. Maybe I had an old version cached in my browser or whatever.

But please squash the commits before this gets merged.

**Comment by ejk43** [comment]
Squashed. Cheers!

**Comment by mbr0wn** [comment]
:+1: good to go!

**Pull request merged**